task3/Makefile

```
@@ -0,0 +1,104 @@
+CС=gcc
+CFLAGS= -Wall -Werror -Wextra
+
+SOURCES=lb1task3.c # Тут вы перечислятете файлы с вашими лабами по 1 файлу на задание
+TEST_TEMPLATES_DIR=/mnt/c/Users/Вероника/Desktop/lb1/test_templates
+EXECUTABLES=$(SOURCES:.c=.a)

```

Поменять на относительную директорию
```suggestion
TEST_TEMPLATES_DIR=../test_templates
```